src/System.Runtime.Numerics/tests/BigInteger/ctor.cs

```diff
@@ -11,390 +11,294 @@ public class BigIntegerConstructorTest
+            VerifyCtorUInt32((UInt32)0);
 
             // ctor(UInt32): 1
-            Assert.True(VerifyCtorUInt32((UInt32)1), " Verification Failed");
+            VerifyCtorUInt32((UInt32)1);
 
             // ctor(UInt32): Int32.MaxValue
-            Assert.True(VerifyCtorUInt32((UInt32)Int32.MaxValue), " Verification Failed");
+            VerifyCtorUInt32((UInt32)Int32.MaxValue);
 
             // ctor(UInt32): Int32.MaxValue+1
-            Assert.True(VerifyCtorUInt32(((UInt32)Int32.MaxValue) + 1), " Verification Failed");
+            VerifyCtorUInt32(((UInt32)Int32.MaxValue) + 1);
 
             // ctor(UInt32): UInt32.MaxValue
-            Assert.True(VerifyCtorUInt32(UInt32.MaxValue), " Verification Failed");
+            VerifyCtorUInt32(UInt32.MaxValue);
 
             // ctor(UInt32): Random Positive
             for (int i = 0; i < s_samples; ++i)
             {
-                Assert.True(VerifyCtorUInt32((UInt32)(UInt32.MaxValue * s_random.NextDouble())), " Verification Failed");
+                VerifyCtorUInt32((UInt32)(UInt32.MaxValue * s_random.NextDouble()));
             }
         }
-        private static bool VerifyCtorUInt32(UInt32 value)
+
+        private static void VerifyCtorUInt32(UInt32 value)
         {
-            bool ret = true;
-            BigInteger bigInteger;
+            BigInteger bigInteger = new BigInteger(value);
 
-            bigInteger = new BigInteger(value);
-
-            if (!bigInteger.Equals(value))
-            {
-                Console.WriteLine("Expected BigInteger {0} to be equal to UInt32 {1}", bigInteger, value);
-                ret = false;
-            }
-            if (String.CompareOrdinal(value.ToString(), bigInteger.ToString()) != 0)
-            {
-                Console.WriteLine("UInt32.ToString() and BigInteger.ToString() on {0} and {1} should be equal", value, bigInteger);
-                ret = false;
-            }
-            if (value != (UInt32)bigInteger)
-            {
-                Console.WriteLine("Expected BigInteger {0} to be equal to UInt32 {1}", bigInteger, value);
-                ret = false;
-            }
+            Assert.True(bigInteger.Equals(value), "Expected BigInteger to be equal to UInt32");
+            Assert.True(String.CompareOrdinal(value.ToString(), bigInteger.ToString()) == 0, "UInt32.ToString() and BigInteger.ToString() should be equal");
+            Assert.True(value == (UInt32)bigInteger, "Expected BigInteger to be equal to UInt32");
 
             if (value != UInt32.MaxValue)
             {
-                if ((UInt32)(value + 1) != (UInt32)(bigInteger + 1))
-                {
-                    Console.WriteLine("Adding 1 to both {0} and {1} should remain equal", value, bigInteger);
-                    ret = false;
-                }
+                Assert.True((UInt32)(value + 1) == (UInt32)(bigInteger + 1), "Adding 1 to both should remain equal");
```

Comment:
Personally, I think the message provides little value, everyone looking at that line should immediately see the intent anyway. Additionally, it's much better in a failed case to get a message like `Expected: 17, Actual: 25` instead of `Expected: True, Actual: False`.